mkt/webapps/tests/test_serializers.py

```diff
@@ -40,6 +42,10 @@ def serialize(self, app, profile=None):
         a = AppSerializer(instance=app, context={'request': self.request})
         return a.data
 
+    def test_base(self):
```

Comment:
Sort of. I thought it was pretty useless to have a lot of different tests for basic fields, so I introduced this method with the hope that future basic tests would go in it :)